Koha/Plugin/EDS/js/EDSScript.tt

```diff
@@ -1236,7 +1275,7 @@ function EDSAutoComp() {
 			console.log("Running creds false...");
 		}
 		jQuery.getJSON('[% PLUGIN_HTTP_PATH %]/opac/eds-ac.pl?type=auth', function (data) {
-			eds_sessionStorage.set('autoComp', data);
+			eds_sessionStorage.set('autoComtranslControl1p', data);
```

Comment:
A quick search of the code shows that this variable doesn't seem to be called anywhere else.  Is this variable causing the autocomplete issues?  It looks like as written, autoComp, which is used elsewhere, never gets set.

Reply:
Changed 'autoComtranslControl1p' to 'autoComp', still dealing with same cart issues.